symfony/config/packages/twig.php

```diff
@@ -15,6 +15,7 @@
         'sessionTimeout' => $env->values['sessionTimeOut'],
         'sessionWarning' => $env->values['sessionWarning'],
         'timeZones' => $env->getTimeZones(),
-        'confluenceResources' => HelpService::$confluenceResources
+        'confluenceResources' => HelpService::$confluenceResources,
+        'orderSampleInfo' => $env->configuration['order_samples_info_3']
```

Comment:
This currently breaks if this configuration item isn't set. Can we add a null coalesce here? Something like:

```suggestion
        'orderSampleInfo' => $env->configuration['order_samples_info_3'] ?? null
```